import_and_export_data/formats/product_model.rst

```diff
@@ -0,0 +1,28 @@
+Product model data structure
+============================
+
+Product models are exported in a file with the following fields:
+
+Attributes fields can be one of these:
+
+- **code** (required): the product model code
+- **clothing_size**: size option code for a simple select option
+- **description-<locale_code>-<channel>**: localizable and scopable attribute for channel mobile one row by local_code dans channel
+- **name-<locale_code>**: localizable attribute one row by local_code
+- **price-EUR**: numeric price value for Euro
+- **price-USD**: numeric price value for US Dollar
```

Comment:
why not the same description than above ? like `**price-<currency>**: numeric price value for this currency (example: price-EUR`